lib/inferno/apps/web/controllers/test_sessions/create.rb

```diff
@@ -3,7 +3,7 @@ module Web
     module Controllers
       module TestSessions
         class Create < Controller
-          PARAMS = [:test_suite_id].freeze
+          PARAMS = [:test_suite_id, :suite_options].freeze
```

Comment:
I'm confused how this works on create, and I don't think it is represented in swagger, is it?  Do I pass in a set of '#{id}=#{value}'?

Reply:
Whoops, I missed that in the swagger. It takes an array of SuiteOptions (basically just hashes with `id` and `value`). There's an [example in the request spec](https://github.com/inferno-framework/inferno-core/blob/fi-1590-suite-options-json-api/spec/requests/test_sessions_spec.rb#L42).